src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -9232,3 +9235,92 @@ definitions:
       id:
         type: string
         description: The unique identifier of the action pending execution
+  Definition:
+    required:
+      - typeId
+    properties:
+      typeId:
+        description: the id of the type
+        type: string
+      title:
+        description: the human-readable class title
+        type: string
+      parentName:
+        description: name of the parent class
+        type: string
+      description:
+        description: description of the class
+        type: string
+      archive:
```

Comment:
I'm not entirely sure what this property signifies to be honest and not sure it would be particularly useful for clients, sounds like a back-end behaviour to me so would should probably err on the side of caution and leave it out, we can always add later.